service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/CaseSubmissionChecker.java

```diff
@@ -49,4 +50,8 @@ public boolean isCompleted(CaseData caseData) {
     public boolean isAvailable(CaseData caseData) {
         return validate(caseData).isEmpty();
     }
+
+    public static List<Event> getRequiredEvents() {
```

Comment:
is this static list for reference? maybe externalising it that could be could be mocked for the test, decoupling the list of events with the logic of the service

Reply:
This is an existing class which is already used for validation. I just exposed the static list to avoid duplication. Happy to create a reference class but it will still be a static list.